tests/unit/conftest.py

```diff
@@ -77,8 +83,10 @@ def engine(pytestconfig):
     echo = pytestconfig.getoption("verbose") > 2
     test_engine = create_engine(test_db_url, echo=echo)
 
-    # TODO: Convert to running alembic migrations
-    Base.metadata.create_all(bind=test_engine)
+    cfg = alembic_config.Config(os.path.join(APP_FOLDER, "alembic.ini"))
+    with test_engine.begin() as cnx:
+        cfg.attributes["connection"] = cnx  # pylint:disable = E1137
```

Comment:
Can we use the long name here (unsupported-assignment-operation) instead of the error code?

Also, is there a way we could avoid using this `pylint:disable` in the first place?

Reply:
> Also, is there a way we could avoid using this pylint:disable in the first place?

Pylint does not understand what is going here:

https://github.com/sqlalchemy/alembic/blob/7a516ebefb8f6bf9041c51aa8711fa2a3a3f4bcd/alembic/config.py#L137-L155

Reply:
I used the error code to avoid wrapping

```diff
-        cfg.attributes["connection"] = cnx  # pylint:disable = E1137
+        cfg.attributes[  # pylint:disable = unsupported-assignment-operation
+            "connection"
+        ] = cnx
```

Reply:
What about 
```python
        # pylint: disable-next=unsupported-assignment-operation
        cfg.attributes["connection"] = cnx
```